ios/browser/api/ads/brave_ads.mm

```diff
@@ -1355,9 +1355,6 @@ - (void)setBooleanPref:(const std::string&)path value:(const bool)value {
 
 - (bool)getBooleanPref:(const std::string&)path {
   const auto key = base::SysUTF8ToNSString(path);
-  if (path == ads::prefs::kShouldAllowConversionTracking) {
-    return [self shouldAllowAdConversionTracking];
```

Comment:
Can you please remove 

```objc
- (BOOL)shouldAllowAdConversionTracking {
  return true;
}
```

As well from this file since its no longer used at all

Reply:
Fixed, thanks